packages/wordpress-external-dependencies-plugin/index.js

```diff
@@ -0,0 +1,23 @@
+const { ExternalsPlugin } = require( 'webpack' );
+
+class WordPressExternalDependenciesPlugin extends ExternalsPlugin {
```

Comment:
`<nitpick>`
The modern JS style is to prefer composition over inheritance:
```js
class MyPlugin {
  constructor() {
    this.innerPlugin = new ExternalsPlugin( 'global', [ wpExternals ] );
  }

  apply( compiler ) {
    this.innerPlugin.apply( compiler );
  }
}
```
One benefit is that it's clear what the plugin's public API is.
`</nitpick>`